scripts/tests/tcp-dns.sh

```diff
@@ -4,8 +4,11 @@ source "./scripts/tests/lib.sh"
 
 client sh -c "apk add bind-tools" # The compat tests run using the production image which doesn't have `dig`.
 
-echo "Resolving DNS resource over TCP"
-client sh -c "dig +tcp dns.httpbin"
+echo "Resolving DNS resource over TCP with search domain"
+client sh -c "dig +tcp dns"
```

Comment:
This is likely relying on the current connlib built-in expansion (which only works for resources and will be removed again). You need to say `+search` for `dig` to use search domains.

Reply:
Ah, thanks. I'll add that.

Reply:
Updated